include/json/writer.h

```diff
@@ -6,6 +6,11 @@
 #ifndef JSON_WRITER_H_INCLUDED
 #define JSON_WRITER_H_INCLUDED
 
+#if defined(_MSC_VER)
+// Disable warning about deprecated destructors
+#pragma warning(disable : 4996)
```

Comment:
Should this have a pragma push and a smaller area of effect?
What specifically are the deprecated destructors? Can they be addressed more directly with a real fix?

Reply:
Beats me. I don't use MSVC.